clients/java/src/test/java/io/camunda/zeebe/client/flownodeinstance/FlownodeInstanceTest.java

```diff
@@ -120,6 +123,18 @@ void shouldSearchFlownodeInstanceWithFullSorting() {
     assertSort(sorts.get(8), "tenantId", "asc");
   }
 
+  @Test
+  public void shouldGetFlownodeInstance() {
+    // when
+    final long flowNodeInstanceKey = 0xC00L;
```

Comment:
🙃 Is there a specific reason for this fancy `long` value? 😅

Reply:
I was a bit bored 😅. And maybe this can be better memorized.